storage/aptosdb/src/state_store/mod.rs

```diff
@@ -448,7 +479,7 @@ impl StateStore {
 
         let latest_snapshot_version = state_db
             .state_merkle_db
-            .get_state_snapshot_version_before(Version::MAX)
+            .get_state_snapshot_version_before(u64::MAX)
```

Comment:
still use Version::MAX?

Reply:
I put an `ensure!` below.

Reply:
? he meant `Version::MAX` instead of `u64::MAX` -- just cosmetic